### PR TITLE
Additional RSpec tests

### LINKED ISSUE #78: (MV) Additional RSpec tests
Create a test for:

NO MANIFEST INVENTORY FILE
NO SIGNATURE CATALOG FILE
INCORRECT_DIR (Should be able to create a druid path with files in the version directory.)
Make sure a version directory with the name of : 'v0001a' doesn't pass

### PR DESCRIPTION
**Pull request by SaravShah**
Create a test for:
NO MANIFEST INVENTORY FILE
NO SIGNATURE CATALOG FILE
INCORRECT_DIR (Should be able to create a druid path with files in the version directory.)
Make sure a version directory with the name of : 'v0001a' doesn't pass

closes #78

### CODE REVIEW

File: spec/unit_tests/moab/storage_object_validator_spec.rb [hunk 1]
```diff
@@ -59,6 +59,21 @@
...
+      druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/cc/000/cc/0000/cc000cc0000'
+      storage_obj = Moab::StorageObject.new(druid, druid_path)
+      storage_obj_validator = described_class.new(storage_obj)
+      verification_array = storage_obj_validator.validation_errors
+      it 'does not have signatureCatalog.xml' do
+        expect(verification_array[0]).to eq(Moab::StorageObjectValidator::NO_SIGNATURE_CATALOG => "Version: v0001 Missing signatureCatalog.xml")
+      end
+
+      it 'does not have manifestInventory.xml' do
+        expect(verification_array[1]).to eq(Moab::StorageObjectValidator::NO_MANIFEST_INVENTORY => "Version: v0002 Missing manifestInventory.xml")
```
Comment (ndushay) [inline spec/unit_tests/moab/storage_object_validator_spec.rb]: this is fine.  Just for future reference, you could write 

```ruby
expect(verification_array).to include(Moab::StorageObjectValidator::NO_MANIFEST_INVENTORY => "Version: v0002 Missing manifestInventory.xml")
```

File: spec/unit_tests/moab/storage_object_validator_spec.rb [hunk 2]
```diff
@@ -67,13 +82,29 @@
...
+    it "has extra characters in version directory name" do
+      druid = 'aa000aa0000'
+      druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/aa/000/aa/0000/aa000aa0000'
+      storage_obj = Moab::StorageObject.new(druid, druid_path)
+      storage_obj_validator = described_class.new(storage_obj)
+      verification_array = storage_obj_validator.validation_errors
+      expect(verification_array[0]).to eq(Moab::StorageObjectValidator::VERSION_DIR_BAD_FORMAT => "Version directory name not in 'v00xx' format")
+    end
+    it "incorrect items in path" do
+      druid = 'aa000aa0000'
```
Comment (jmartin-sul) [inline spec/unit_tests/moab/storage_object_validator_spec.rb]: should this be `'bb000bb0000'`?

Comment (SaravShah) [reply spec/unit_tests/moab/storage_object_validator_spec.rb]: oh yeah. oops

File: spec/unit_tests/moab/storage_object_validator_spec.rb (hunk 1 above)
Comment (SaravShah) [reply spec/unit_tests/moab/storage_object_validator_spec.rb]: ah okay makes sense

Comment (ndushay) [reply spec/unit_tests/moab/storage_object_validator_spec.rb]: stating the obvious:  it makes sense to do it your way if you care about which position in the array has the match.

Comment (SaravShah) [reply spec/unit_tests/moab/storage_object_validator_spec.rb]: Okay - i'll update my code

**Comment by SaravShah** [comment]
I changed some of the specs to ```.to include ``` but left the ones i thought where the position matter as ``` (array[0]).to eq ```

**Pull request merged**